src/helper_functions.py

```diff
@@ -0,0 +1,135 @@
+import sys
+import os
+import re
+import constants
+import subprocess
+import pytz
+import requests
+import json
+
+from datetime import datetime, timedelta, timezone
+from osmeterium.run_command import run_command
+from constants import log, process_log
+
+def get_current_datetime():
+    return datetime.now(tz=timezone.utc)
+
+def datetime_to_string(datetime):
+    return datetime.strftime(r'%Y-%m-%dT%H:%M:%SZ')
+
+def string_to_datetime(string):
+    datetime_str = datetime.strptime(string, r'%Y-%m-%dT%H:%M:%SZ')
+    return datetime_to_utc(datetime_str)
+
+def datetime_to_utc(datetime):
+    time_zone_dt = pytz.timezone('UTC').localize(datetime)
+    return time_zone_dt.astimezone(pytz.utc)
+
+def handle_subprocess_exit_code(exit_code):
+    log.error(fr'The subprocess raised an error: {exit_code}')
+    raise
+
+def log_and_exit(exception_message):
+    log.error(exception_message)
+    sys.exit(1)
+
+def subprocess_get_stdout_output(args, remove_new_lines=True):
+    completed_process = subprocess.run(args=args,
+                                        stdout=subprocess.PIPE,
+                                        stderr=subprocess.PIPE,
+                                        universal_newlines=True)
+                                        
+    if completed_process.returncode is not 0:
+        raise
+    output = completed_process.stdout
+    if remove_new_lines: 
+        return output.replace('\n', '')
+    return output
```

Comment:
if a subprocess has different log lines this function still concats them to 1 message and logs it, is it the desired behavior?

Reply:
using the strip function.